include/labwc.h

```diff
@@ -504,7 +504,7 @@ void seat_output_layout_changed(struct seat *seat);
  * geometry->{width,height} are provided by the caller.
  * geometry->{x,y} are computed by this function.
  *
- * @note When drag-resistance is used, cursor_x/y should be the original
+ * @note When unsnap-resistance is used, cursor_x/y should be the original
```

Comment:
OK. I'll fix it.

I thought "resistance" describes its meaning more clearly, but I agree that just using `unsnap-threshold` is better to avoid confusion.